patches/server/0952-fixed-null-pointer-exception-when-invoking-Allay-sto.patch

```diff
@@ -0,0 +1,25 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: NonSwag <mrminecraft00@gmail.com>
+Date: Sun, 4 Dec 2022 13:58:15 +0100
+Subject: [PATCH] fixed null pointer exception when invoking Allay#stopDancing
```

Comment:
nitpick, but can changed to "fixed NPE ..." to fit more text into the patch file name